dev-docs/pg-post-upgrade-steps.md

```diff
@@ -0,0 +1,46 @@
+### Follow these steps to upgrade a PostgreSQL major version
```

Comment:
Since this upgrade changes attributes and requires one time manual processes before the upgrade we need to consider it to be a major version upgrade. We should make this upgrade the 5.0 release so we properly communicate the work involved to our customers.

Reply:
We can remove this file as we have included all the steps in the `supermarket_upgrade.md` file. Will remove this file.